### PR TITLE
feat: empty state within the dashboard to help guide users how to cre…

### PR DESCRIPTION
**Pull request by Priyanshu44**
image

Changes in PR:

->Added empty state for dashboard in the centre of the dashboard.
->Add test cases to ensure empty state is present for an empty dashboard. (Skipped it for now like other test suites in the file as we have to likely need to mock TwinMaker client)

### CODE REVIEW

File: packages/dashboard/src/components/internalDashboard/dashboardEmptyState.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  top: 50%;
+  right: 50%;
+  border: dashed;
+  height: 200px;
+  margin-top: -100px;
+  width: 350px;
+  margin-right: -125px;
+  text-align: center;
+  font-size: 16px !important;
+  font-weight: 600 !important;
```
Comment (tracy-french) [inline packages/dashboard/src/components/internalDashboard/dashboardEmptyState.css:12]: Is it possible to avoid using `!important`?

File: packages/dashboard/src/components/internalDashboard/index.test.tsx [hunk 2]
```diff
@@ -130,3 +135,34 @@ it.skip('toggles to preview mode and hides the component library', function () {
   expect(screen.queryByText(/time machine/i)).toBeInTheDocument();
   expect(screen.queryByText(/actions/i)).toBeInTheDocument();
 });
+
+// TODO: fix these tests (likely need to mock TwinMaker client)
```
Comment (tracy-french) [inline packages/dashboard/src/components/internalDashboard/index.test.tsx:139]: What's preventing us from doing this in this PR?
